### ISSUE TITLE
Oauth & Personal access tokens

### ISSUE THREAD

**Issue by aarontanzk** [saqueib/qtube#3]
Hi i'm new with Passport. I see the user being prompted to create Oauth & Personal Access Token at dashboard.

Any idea how to resolve this?

**Comment by aarontanzk** [comment]
and the API like api/channels returns a json response with no views. are repo users supposed to create their own views?

**Comment by saqueib** [comment]
@aarontanzk yes, `api/any-thing` will give you a response in JSON, you can use the same in your vue frontend.

**Comment by aarontanzk** [comment]
Hi , thanks for the fast response.

this repo doesn't include the vue frontend?

**Comment by saqueib** [comment]
http://www.qcode.in/youtube-like-app-vue-js-laravel/ check this, click on source code for front-end code

**Comment by aarontanzk** [comment]
clicking on the source code brings me back to here.

https://github.com/saqueib/qtube/tree/v1.0


am I supposed to use the source code + tutorial or is there a full version?


many thanks.
